server/leaderboard/urls.py

```diff
@@ -8,4 +8,7 @@
         name="leetcode-leaderboard",
     ),
     path("github/", views.GitHubLeaderboardView.as_view(), name="github-leaderboard"),
+    path("internship/", views.InternshipApplicationLeaderboardView.as_view(), name="internship-leaderboard"),
+    path("newgrad/", views.NewGradApplicationLeaderboardView.as_view(), name="newgrad-leaderboard"),
```

Comment:
Can you change `newgrad` to `new-grad`? The linter complains over on the frontend lol.